cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -55,6 +55,7 @@
     public static final String ENV_VAR_ZOOKEEPER_SESSION_TIMEOUT_MS = "STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS";
     public static final String ENV_VAR_TOPIC_METADATA_MAX_ATTEMPTS = "STRIMZI_TOPIC_METADATA_MAX_ATTEMPTS";
     public static final String ENV_VAR_TLS_ENABLED = "STRIMZI_TLS_ENABLED";
+    public static final String ENV_VAR_TLS_AUTH_ENABLED = "STRIMZI_TLS_AUTH_ENABLED";
```

Comment:
I guess you will need to also use the new variable here? https://github.com/strimzi/strimzi-kafka-operator/blob/f2ce3ea17c28846ad61640df55f46231a4cea17f/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java#L268

Reply:
Thanks for noticing!
Fixed.